packages/components/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 7.5.0
```

Comment:
It might be 8.0.0 as well so it's safe to put and:
```suggestion
## Master
```
and update after release :)